Add contribution guide (#96)
1Password/op-scim-helm: CONTRIBUTING.md

## How to contribute

Thank you for taking the time to contribute to the Helm chart for the 1Password SCIM bridge.

#### **Did you find a bug?**

> 

Reviewer: Perhaps these should all be an ordered list, instead of unordered.

Author: I went back and forth on whether these lists should be ordered or unordered and ultimately kept it unordered for consistency.

Also, I quite like the way the Rails team did [theirs](https://github.com/rails/rails/blob/main/CONTRIBUTING.md), which I used as a starting point for this.

> * Do not open up a GitHub issue if the bug is an issue with the 1Password SCIM bridge or functionality related to automated provisioning, and instead reach out to the our [support team](https://support.1password.com/contact/) directly.

Reviewer: ```suggestion
* If the bug an issue with the 1Password SCIM bridge or functionality related to automated provisioning in 1Password, do not open up a GitHub issue. Instead reach out to the our [support team](https://support.1password.com/contact/) directly.
```

> * Ensure the bug was not already reported by searching on GitHub under [Issues](https://github.com/1Password/op-scim-helm/issues).

Reviewer: ```suggestion
* Search if the bug has already been reported by searching on GitHub under [Issues](https://github.com/1Password/op-scim-helm/issues). Comment on the issues with your context to help us resolve the issue.
```

* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/1Password/op-scim-helm/issues/new). Be sure to include a title and clear description, as much relevant information as possible, and steps that demonstrate the expected behavior that is not occurring.

#### **Did you write a patch that fixes a bug?**

* Open a new GitHub pull request with the patch.

* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.

#### **Did you fix whitespace, formatting, or make a purely cosmetic patch?**

> Changes that are cosmetic in nature and do not add anything substantial to the stability, functionality, or testability of this project will generally not be accepted.

Reviewer: ```suggestion
Changes that are cosmetic in nature and do not add anything substantial to the stability, functionality, or testability of this project will be rejected.
```

* Before submitting a PR with suggested changes, consider filing a [new issue](https://github.com/1Password/op-scim-helm/issues/new) that describes the problem, how it could be addressed and how this is an improvement over the status quo.

* Suggest your change by subitting a new [PR](https://github.com/1Password/op-scim-helm/compare). Be sure to include a detailed description of the changes, the problem that's being addressed and how this is objectively a worthwhile addition to the project.

#### **Do you have questions about the SCIM bridge or automated provisioning?**

* Find more information on our [support portal](https://support.1password.com/scim/).

* Ask a question on our [community support forum](https://1password.community/categories/scim-bridge).

* Contact our [support team](https://support.1password.com/contact/).

Thanks! :heart:

> Provisioning Team @ [1Password](https://1password.com)

Reviewer: ```suggestion
The Provisioning Team @ [1Password](https://1password.com)
```
